### ISSUE TITLE
Uncaught TypeError: Super expression must either be null or a function

### ISSUE THREAD

**Issue by vincent-the** [react-native-material-design/react-native-material-design#69]
I receive the following error:
`Uncaught TypeError: Super expression must either be null or a function, not undefined `

![bildschirmfoto vom 2016-05-23 23 10 53](https://cloud.githubusercontent.com/assets/4801522/15484882/89a93976-213c-11e6-9444-71871d600071.png)

My index.android.js:
```
import React, { Component } from 'react';
import {
  AppRegistry,
  StyleSheet,
  Text,
  View
} from 'react-native';
import { Card, Button} from 'react-native-material-design';

class app extends Component {
  render() {
    return (
      <View style={styles.container}>
        <Text style={styles.welcome}>
          Welcome to React Native!
        </Text>
        <Text style={styles.instructions}>
          To get started, edit index.android.js
        </Text>
        <Text style={styles.instructions}>
          Shake or press menu button for dev menu
        </Text>
      </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
    backgroundColor: '#F5FCFF',
  },
  welcome: {
    fontSize: 20,
    textAlign: 'center',
    margin: 10,
  },
  instructions: {
    textAlign: 'center',
    color: '#333333',
    marginBottom: 5,
  },
});

AppRegistry.registerComponent('app', () => app);

```

**Comment by little51** [comment]
the last version on github is ok. clone react-native-material-design to node_modules,and in react-native-material-design forder,execute npm install.

**Comment by vincent-the** [comment]
@little51 Thx, it works

**Comment by Ehesp** [comment]
I'll push up a new version tonight

**Comment by Ehesp** [comment]
Should now be fixed in 0.3.6... Cheers
